data/mapcss-mapping.csv

```diff
@@ -19,7 +19,7 @@ natural|tree;18;
 place|locality;19;
 natural|coastline;20;
 place|village;21;
-deprecated|deprecated;22;x
+boundary|military;22;
```

Comment:
Please, put this and landuse|military|danger_area somewhere down, near landuse|military say 1000 line, as rare tags and keep this top numbers for the future more common tags.

Reply:
Done